### ISSUE TITLE
Improve contrast of series/video selection and add missing form label

### ISSUE THREAD

**Issue by owi92** [elan-ev/tobira#731]
The wave plugin is complaining about the contrast of the "select option..." placeholder and a missing form label in the new select fields for series/video selection in "edit page content" and also the video uploader.

**Comment by owi92** [comment]
Hmm the `missing form label` seems to be an issue with `react select` and I don't think this can be fixed in Tobira.
On the other hand, the contrast issue appears to have been fixed somewhere along the way.
So I think we can close this for now.
